automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala

```diff
@@ -163,125 +163,150 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
+          val configName = methodName + "Config"
+          api.methodConfigurations.createMethodConfigInWorkspace(billingProject, workspaceName, MethodData.SimpleMethod.copy(methodName = methodName),
+            SimpleMethodConfig.configNamespace, configName, 1, SimpleMethodConfig.inputs, SimpleMethodConfig.outputs, "participant")
+          api.methods.redact(MethodData.SimpleMethod.copy(methodName = methodName))
 
+          signIn(uiUser)
+          val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, SimpleMethodConfig.configNamespace, configName).open
           methodConfigDetailsPage.ui.openEditMode()
-          methodConfigDetailsPage.ui.saveEdits("disabled")
-          val messageModal = MessageModal()
-          assert(messageModal.validateLocation)
-          messageModal.clickCancel()
+          methodConfigDetailsPage.ui.checkSaveButtonState shouldEqual "disabled"
+          methodConfigDetailsPage.ui.clickSaveButton()
+          val modal = MessageModal()
+          modal.validateLocation shouldBe true
+          modal.clickCancel()
 
           methodConfigDetailsPage.ui.changeSnapshotId(2)
-          methodConfigDetailsPage.ui.saveEdits()
-          assert(!methodConfigDetailsPage.ui.isSnapshotRedacted())
+          methodConfigDetailsPage.ui.clickSaveButton()
+          methodConfigDetailsPage.ui.isSnapshotRedacted() shouldBe false
         }
       }
     }
-    "launch analysis button should be disabled and show error if clicked" in withWebDriver { implicit driver =>
+    "should have warning icon in config list" in withWebDriver { implicit driver =>
       withWorkspace(billingProject, "MethodConfigTabSpec_redacted_launch_analysis_error") { workspaceName =>
-        withConfigForRedactedMethodInWorkspace("MethodConfigTabSpec_redacted_launch_analysis_error", billingProject, workspaceName, true) { configName =>
+        withMethod("MethodConfigTabSpec_redacted_has_warning_icon", MethodData.SimpleMethod) { methodName =>
+          val configName = methodName + "Config"
+          api.methodConfigurations.createMethodConfigInWorkspace(billingProject, workspaceName, MethodData.SimpleMethod.copy(methodName = methodName),
+            SimpleMethodConfig.configNamespace, configName, 1, SimpleMethodConfig.inputs, SimpleMethodConfig.outputs, "participant")
+
+          api.methods.redact(MethodData.SimpleMethod.copy(methodName = methodName))
+
           signIn(uiUser)
-          val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, MethodData.SimpleMethodConfig.configNamespace, configName).open
-          assert(methodConfigDetailsPage.ui.isSnapshotRedacted())
+          val workspaceMethodConfigPage = new WorkspaceMethodConfigListPage(billingProject, workspaceName).open
+          workspaceMethodConfigPage.filter(configName)
+          find(cssClass(s"fa-exclamation-triangle fa ")).isDefined shouldBe true
```

Comment:
Specifics for locating elements (in this case by class names) should be in the page class. Also, it would be good for the page class return one of a possible set of states to check against instead of a boolean.